### PR TITLE
Exercise 2

### PR DESCRIPTION
**Pull request by iret-jez**
First commit is a simple addition of assert statement to verify if the login was successful. With one more selector and helpeer function, that I found to be useful. 

The second commit is a test for adding the item to a cart with selectors and helpers and new files. 

Also, I've added pytest import in order to run tests with a simple `pytest` command. 
I completely don't like the amount of duplicate code, but I wasn't able to get rid of it without using classes and maybe Pytest fixtures. I need to dive into it if you'll find it necessary on this stage.

### CODE REVIEW

File: helpers/helpers.py [hunk 1]
```diff
@@ -18,3 +19,15 @@ def type_text_to_input(driver, text, selector):
 def click_on_element(driver, selector):
     WebDriverWait(driver, 15).until(EC.visibility_of_element_located((By.XPATH, selector)))
     driver.find_element(By.XPATH, selector).click()
+
+
+def check_if_element_exists(driver, selector):
```
Comment (WaclawZurko) [inline helpers/helpers.py:36]: GJ, nice one

File: tests/shop_page_test.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+
+browser = webdriver.Chrome()
+browser.get(login_page)
+
+type_text_to_input(browser, "standard_user", USERNAME_XPATH)
+type_text_to_input(browser, "secret_sauce", PASSWORD_XPATH)
+click_on_element(browser, LOGIN_BUTTON_XPATH)
+
+time.sleep(3)
```
Comment (WaclawZurko) [inline tests/shop_page_test.py:21]: We will learn how to omit "hard sleeps" but for now its fine

File: tests/shop_page_test.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+click_on_element(browser, ADD_TO_CART_SECOND_ITEM_XPATH)
+
+def test_item_added_to_cart():
```
Comment (WaclawZurko) [inline tests/shop_page_test.py]: Its just a declaration of function - we need also "call" this function to make it run.
If this function is specific only for "shop page" maybe its nice to move it to pages/shop_page.py?

**Review by WaclawZurko — CHANGES_REQUESTED** [review]
Hey, please:
+ move function "test_item_added_to_cart" to pages/shop_page.py
+ call that function in "tests/shop_page_test.py"

**Comment by iret-jez** [comment]
First commit - move declaration of *_test.py functions from _test.py files. Add .vscode.settings.json file in order VSCode can resolve the paths easier and highlight the syntax correctly. 

Second commit - add headless mode and move options and driver initialization into helpers.py

**Review by WaclawZurko — APPROVED** [review]

**Pull request merged**